src/cinder/GeomIo.cpp

```diff
@@ -462,36 +462,22 @@ void Target::generateIndices( Primitive sourcePrimitive, size_t sourceNumIndices
 const float Rect::sNormals[4*3] = {0, 0, 1,	0, 0, 1,	0, 0, 1,	0, 0, 1 };
 const float Rect::sTangents[4*3] = {0.7071067f, 0.7071067f, 0,	0.7071067f, 0.7071067f, 0,	0.7071067f, 0.7071067f, 0,	0.7071067f, 0.7071067f, 0 };
 
-Rect::Rect()
+Rect::Rect() : mHasColors( false )
```

Comment:
spacing: according to the [styleguide](https://github.com/cinder/Cinder/blob/glNext/CONTRIBUTING.md#example-class-implementation), should be:

```
Rect::Rect()
    : mHasColors( false )
{
...
```